code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -214,13 +214,34 @@
 		icon_state = "[initial(icon_state)]0"
 
 /obj/item/reagent_containers/hypospray/medipen/survival
-	name = "survival medipen"
-	desc = "A medipen for surviving in the harshest of environments, heals and protects from environmental hazards. WARNING: Do not inject more than one pen in quick succession."
+	name = "survival emergency medipen"
+	desc = "A medipen for surviving in the harsh environments, heals most common damage sources. WARNING: May cause organ damage."
 	icon_state = "stimpen"
 	inhand_icon_state = "stimpen"
+	volume = 30
+	amount_per_transfer_from_this = 30
+	list_reagents = list( /datum/reagent/medicine/epinephrine = 8, /datum/reagent/medicine/c2/aiuri = 8, /datum/reagent/medicine/c2/libital = 8 ,/datum/reagent/medicine/leporazine = 6)
+
+/obj/item/reagent_containers/hypospray/medipen/survival/inject(mob/living/M, mob/user)
+	if(!lavaland_equipment_pressure_check(get_turf(user)))
+		if(M in user.do_afters)
+			to_chat(user,"<span class='notice'>You are already releasing the low-pressure gauge!</span>")

```

Comment:
```suggestion
			to_chat(user,"<span class='notice'>You are too busy to use [src]!</span>")
```

1. doafters are not specific to the pen

2. Why does the suggestion feature automatically generate a new line? who did this???